packages/stories-block/src/block/components/authorSelection.js

```diff
@@ -107,29 +119,17 @@ const AuthorSelection = ({ authors: authorIds, setAttributes }) => {
    * @param {string} search Search query to look for authors.
    * @return {void}
    */
-  const onInputChange = (search) => {
-    apiFetch({
-      path: addQueryArgs('/wp/v2/users', { per_page: 100, search }),
-    })
-      .then((users) => {
-        if ('undefined' !== typeof users && Array.isArray(users)) {
-          setAuthorSuggestions(
-            users.map(({ id, name }) => ({ id, value: name }))
-          );
-        }
-      })
-      .catch(() => {
-        setAuthorSuggestions([]);
-      });
-  };
+  const onInputChange = (search) => setAuthorKeyword(search);
 
   const debouncedOnInputChange = useDebounce(onInputChange, 500);
 
   return (
     <Autocomplete
       label={__('Authors', 'web-stories')}
-      value={authorsList.map(({ value }) => value)}
-      options={authorSuggestions.map(({ value }) => value)}
+      value={authorsList ? authorsList.map(({ name }) => name) : []}
```

Comment:
This check is not needed because `authorsList` is always an array (because of the `authorsList: getUsers(query) || [],` above)